packages/xo-web/src/xo-app/vm/tab-network.js

```diff
@@ -710,7 +710,12 @@ const COLUMNS = [
       <VifNetwork vif={vif} network={networks[vif.$network]} resourceSet={resourceSet} />
     ),
     name: _('vifNetworkLabel'),
-    sortCriteria: (vif, userData) => userData.networks[vif.$network].name_label,
+    sortCriteria: (vif, userData) => {
+      if (isEmpty(userData.ip)) {
```

Comment:
`userData.networks`? Because the ip is not used.

You can also use `@xen-orchestra/defined#get` to ignore errors if one nested property is `undefined`.